packages/pipeline-editor/src/properties.json

```diff
@@ -18,6 +18,21 @@
       "enum": [],
       "required": true
     },
+    {
+      "id": "cpu",
+      "type": "double",
+      "required": "false"
+    },
+    {
+      "id": "gpu",
+      "type": "double",
```

Comment:
do we need to set this as integer?

Reply:
In the future we will, but right now changing it to integer won't change any behaviors in canvas